tests/nodeos_run_test.py

```diff
@@ -65,7 +69,8 @@
 
         abs_path = os.path.abspath(os.getcwd() + '/unittests/contracts/eosio.token/eosio.token.abi')
         traceNodeosArgs=" --http-max-response-time-ms 990000 --trace-rpc-abi eosio.token=" + abs_path
-        if cluster.launch(prodCount=prodCount, onlyBios=onlyBios, dontBootstrap=dontBootstrap, extraNodeosArgs=traceNodeosArgs) is False:
+        specificNodeosInstances={0: "bin/nodeos"}
```

Comment:
Why is this needed?

Reply:
It's the only test of the specific nodeos instances feature.  The value is redundant because that's all we have in a normal build, but the feature exists to allow specifying a binary.  The feature existed in `eosio-launcher` but was never tested.